ab_tool/templates/ab_tool/experiments_dashboard.html

```diff
@@ -90,7 +90,7 @@ <h3 class="panel-title panel-title-experiment">Experiment "{{experiment.name}}"<
                         </a>
                     </div>
                     <div class="exp-delete" data-toggle="tooltip" data-placement="left" title="Delete experiment">
-                        <div class="arrow-right"></div>
+                        <div class="arrow-right" id="{{experiment.name}}"></div>
```

Comment:
1. Keep in mind the `id` must be unique for every DOM element in the page; so while `experiment.name` is meant to be unique (so it's a good start), it's possible that some other element might want to do something similar with the name. I think changing this to e.g. `id="delete-{{experiment.name}}"` puts a 'scope' to the id so that it's easier to understand when viewing in the code. (A locator searching for `experiment.name` by `id` might be ambiguous -- viewing that locator, I have no idea what kind of element I might be locating, or what function that element might have.)
2. If this is for testing, are you sure you want the locator `id` on the parent div of the actual delete link (the `a` element)? Seems like we want the `id` of the `a` element so we can locate it directly and click on it.